src/test/java/io/anserini/collection/CoreDocumentTest.java

```diff
@@ -46,20 +54,16 @@ public void setUp() throws Exception {
       "  \"abstract\": \"this is the abstract 1\"," +
       "  \"topics\": [\"Topic 1\", \"Other\"]," +
       "  \"year\": 2020," +
-      "  \"field1\": \"doc1 field1 content\"," +
-      "  \"field2\": \"doc1 field2 content\"," +
-      "  \"nested_field\": {\"doc1key1\": \"doc1value1\", \"doc1key2\": \"doc1value2\"}" +
+      "  \"nested_field\": {\"doc1key1\": \"doc1value1\"}" +
       "}\n" +
```

Comment:
it would be better if we could add test cases for deeper and broader nested fields like `"enrichments": {"references": [], "documentType": {"type": null, "confidence": null}}`

Reply:
Oops you're right I forgot to move some of the tests over, will add a new CoreGeneratorTest for JSON node parsing